pkg/skaffold/render/renderer/helm/helm.go

```diff
@@ -154,17 +154,6 @@ func (h Helm) generateHelmManifest(ctx context.Context, builds []graph.Artifact,
 		return nil, helm.UserErr(fmt.Sprintf("cannot expand chart path %q", release.ChartPath), err)
 	}
 
```

Comment:
it'll be better to write tests for the `generateHelmManifest` and refactor it after

Reply:
yes, it was a good idea to extract them and test independently, but at the same time, you've changed the logic as I've explained here https://github.com/GoogleContainerTools/skaffold/pull/9696#discussion_r1937391395
let's wait for the mainainers)